config/go.d/couchdb.conf

```diff
@@ -0,0 +1,173 @@
+# Each source will have its own set of charts.
+#
+# IMPORTANT:
+#  - Parameter 'name' is mandatory.
+#  - Jobs with the same name are mutually exclusive. Only one of them will be allowed running at any time.
+#
+# This allows autodetection to try several alternatives and pick the one that works.
+# Any number of jobs is supported.
+#
+# The JOBS section format:
+#
+# jobs:
+#   - name: job1
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#
+#
+# [ List of JOB specific parameters ]:
+#  - url
+#    Server URL.
+#    Syntax:
+#      url: http://localhost:5984
+#
+#  - username
+#    Username for basic HTTP authentication.
+#    Syntax:
+#      username: tony
+#
+#  - password
+#    Password for basic HTTP authentication.
+#    Syntax:
+#      password: stark
+#
+#  - proxy_url
+#    Proxy URL.
+#    Syntax:
+#      proxy_url: http://localhost:3128
+#
+#  - proxy_username
+#    Username for proxy basic HTTP authentication.
+#    Syntax:
+#      username: bruce
+#
+#  - proxy_password
+#    Password for proxy basic HTTP authentication.
+#    Syntax:
+#      username: wayne
+#
+#  - timeout
+#    HTTP response timeout.
+#    Syntax:
+#      timeout: 5
+#
+#  - method
+#    HTTP request method.
+#    Syntax:
+#      method: GET
+#
+#  - body
+#    HTTP request method.
+#    Syntax:
+#      body: '{fake: data}'
+#
+#  - headers
+#    HTTP request headers.
+#    Syntax:
+#      headers:
+#        X-API-Key: key
+#
+#  - not_follow_redirects
+#    Whether to not follow redirects from the server.
+#    Syntax:
+#      not_follow_redirects: yes/no
+#
+#  - tls_skip_verify
+#    Whether to skip verifying server's certificate chain and hostname.
+#    Syntax:
+#      tls_skip_verify: yes/no
+#
+#  - tls_ca
+#    Certificate authority that client use when verifying server certificates.
+#    Syntax:
+#      tls_ca: path/to/ca.pem
+#
+#  - tls_cert
+#    Client tls certificate.
+#    Syntax:
+#      tls_cert: path/to/cert.pem
+#
+#  - tls_key
+#    Client tls key.
+#    Syntax:
+#      tls_key: path/to/key.pem
+#
+#  - node
+#    CouchDB node name. Same as -name vm.args argument.
+#    Syntax:
+#      node: node@host
+#
+#  - databases
+#    List of database names for which db-specific stats should be displayed.
+#    Syntax:
+#      databases: plantdb animaldb
+#
+#
+# [ JOB defaults ]:
+#  url: http://127.0.0.1:5984
+#  node: _local
+#  timeout: 5
+#  method: GET
+#  not_follow_redirects: no
+#  tls_skip_verify: no
+#
+#
+# [ JOB mandatory parameters ]:
+#  - name
+#  - url
+#
+# ------------------------------------------------MODULE-CONFIGURATION--------------------------------------------------
+# [ GLOBAL ]
+# update_every: 10
+# autodetection_retry: 0
+# priority: 70000
+#
+#
+# [ JOBS ]
+# jobs:
+#   - name: local
+#     url: http://127.0.0.1:5984
+#     node: node@host
+#     username: admin
+#     password: password
+#     databases: my-db
```

Comment:
Is there default username/pass? Is it possible to gather data with auth?

Reply:
No and no.

Reply:
ok, i was hoping we could have the job uncommented